src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -179,17 +178,21 @@ def configure_container_dict(
     else:
         logger.debug(f"container_dict before processing: {container_dict}")
 
-    if host_mount_path and "volumes" in container_dict:
+    if "volumes" in container_dict:
         logger.warning(
             "'volumes' keyword specified in 'container_dict', but "
             "it is going to be overwritten by specified 'host_mount_path'."
         )
         container_dict.pop("volumes")
 
-    if host_mount_path and not os.path.exists(host_mount_path):
-        Path(host_mount_path).mkdir(parents=True, exist_ok=True)
+    if not os.path.exists(host_mount_path):
+        os.makedirs(host_mount_path)
 
-    if container_mount_path and "volumes" in container_dict:
+    if not container_mount_path:
+        container_mount_path = os.getenv(
+            "PYFLUENT_CONTAINER_MOUNT_PATH", DEFAULT_CONTAINER_MOUNT_PATH
+        )
+    elif "volumes" in container_dict:
         logger.warning(
             "'volumes' keyword specified in 'container_dict', but "
```

Comment:
```suggestion
            "'volumes' keyword is specified in 'container_dict', but "
```
The string needs to have a verb.